react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,36 @@
 import { isArray } from './utils';
 
 export function min() {
-  isArray();
-  return Math.min();
+  if(arguments.length === 0){
+    return undefined;
+  }
+
+  let numbers = arguments[0];
```

Comment:
The implementation is fine, but the idea of this excercise is that you get the arguments of the function using the  spread operator! You can find more info here: https://davidwalsh.name/spread-operator